item.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :genre, :author, :label
```

Comment:
- [ ] Based on the requirements, kindly make Genre, Author, and Label, in your item class to be read-only 👍🏿 

```
attr_reader :genre
attr_reader :author
attr_reader :label
```

Reply:
@prolajumokeoni noted